test/system/test_ppmSeq_qc_analysis.py

```diff
@@ -33,13 +33,13 @@ def test_balanced_strand_analysis_ppmSeq_v1(tmpdir):
             "--output-path",
             tmpdir.dirname,
             "--output-basename",
-            "balanced_strand.healthy.022782-Lb_2146-UGAv3-168",
+            "ppmSeq.healthy.022782-Lb_2146-UGAv3-168",
             "--legacy-histogram-column-names",
         ]
     )
 
 
-def test_balanced_strand_analysis_ppmSeq_v2(tmpdir):
+def test_ppmSeq_analysis_ppmSeq_v2(tmpdir):
```

Comment:
what's v2? now we have only legacy_v5 and v1

Reply:
You might be seeing an outdated version, it's not named v2 anymore

Reply:
Changed to _v1 in any case